src/Orchard.Web/Modules/Orchard.OutputCache/Migrations.cs

```diff
@@ -56,5 +65,17 @@ public class Migrations : DataMigrationImpl {
 
             return 7;
         }
+
+        public int UpdateFrom7() {
+            var cacheSettings = _orchardServices.WorkContext.CurrentSite.As<CacheSettingsPart>();
+            if (cacheSettings.VaryByQueryStringParameters != null) {
```

Comment:
Might want to do a `String.IsNullOrWhitespace()` rather than a simple null check on the settings property.